__tests__/commands/docs/upload.test.ts

```diff
@@ -322,7 +322,7 @@ describe('rdme docs upload', () => {
       afterEach(githubActionsEnv.after);
 
       it('should create a guides page in ReadMe and include `x-readme-source-url` source header', async () => {
-        const headMock = getAPIv2MockForGHA({ authorization }).get('/versions/stable/guides/new-doc').reply(404);
```

Comment:
i wonder if moving these calls from HEAD to GET is what is causing the slowdowns we've been seeing during upload.

Reply:
maybe? i'm pretty sure it hits the same code path since our server doesn't have `HEAD` endpoints defined for guides